installing_on_kubernetes/_topics/migrate_from_appliance.md

```diff
@@ -1,5 +1,11 @@
 ## Migrating from Appliances
 
+### Notes
+- Current architecture in podified limits us to running a single server and zone in a Kubernetes namespace.
```

Comment:
I thought that we would leverage Kubernetes node labels to allow for multiple zones.  Or what that work never completed?  /cc @Fryguy

Reply:
I believe that is the design idea, but I don't think it is implemented yet.